python/ray/data/dataset.py

```diff
@@ -1336,9 +1211,6 @@ def sort(self,
             >>> # Sort by a key function.
             >>> ds.sort(lambda record: record["field1"] % 100)
 
-            >>> # Sort by multiple columns (not yet supported).
-            >>> ds.sort([("field1", "ascending"), ("field2", "descending")])
```

Comment:
So we decided to not support `[(key1, order1), (key2, order2)]` syntax but in favor of `sort([key1, key2], descending=[true, false])`?

Reply:
It's not supported yet, so I removed it (doesn't preclude adding it in the future).